tests/run-make/cross-lang-lto-clang/rmake.rs

```diff
@@ -0,0 +1,61 @@
+// This test checks that cross-language inlining actually works by checking
+// the generated machine code.
+// See https://github.com/rust-lang/rust/pull/57514
+
+//@ needs-force-clang-based-tests
+// FIXME(#126180): This test doesn't actually run anywhere, because the only
+// CI job that sets RUSTBUILD_FORCE_CLANG_BASED_TESTS runs very few tests.
```

Comment:
Suggestion: change `FIXME` -> `NOTE`, and say that this is only (at the time of writing) exercised on `x86_64-gnu-debug` because that CI job sets `RUSTBUILD_FORCE_CLANG_BASED_TESTS`.